### PR TITLE
UI enhancement

### PR DESCRIPTION
**Pull request by Gautam-Hegde**
## Summary of changes
- Added `Material-UI`(MUI) dependency
- Styled text input and button
- Applied a dark mode background color with a linear gradient for stat card `linear-gradient(45deg, #0099f7, #f11712)`
- Added loading indicator with `Skeleton`: Implemented a loading indicator using Material-UI's Skeleton component to provide a better user experience while data is being fetched. The Skeletons mimic the structure of the final content, making the loading process visually appealing.
- Added fallback pfp if user doesn't have a profile picture

## Results

Screenshot (230)
### Loading Skeleton
Screenshot (231)

### CODE REVIEW

**Comment by Jiganesh** [comment]
Hey @Gautam-Hegde, Can you use some solid colors ? greyish black card type ? would look great.

Appreciate the effort pulled !!

File: pages/index.js [hunk 1]
```diff
@@ -1,14 +1,57 @@
+import { Button, Card, CardContent, CardMedia, TextField, Typography, Skeleton } from "@mui/material";
 import { useState } from "react";
 
 const NOT_AVAILABLE_LABEL = "NOT AVAILABLE"
 
+const styles = {
```
Comment (Jiganesh) [inline pages/index.js]: please move this to globals.css

**Comment by Gautam-Hegde** [comment]
> Hey @Gautam-Hegde, Can you use some solid colors ? greyish black card type ? would look great.
> 
> Appreciate the effort pulled !!
- Used UI-flare border shadow to card 
- Embossed avatar
- Simple hover transition🚀 

Screenshot (232)

**Comment by Gautam-Hegde** [comment]
Unexpected UI behaviour of the card component in the production.

**Pull request merged**